pom.xml

```diff
@@ -21,13 +21,6 @@
         <tag>HEAD</tag>
     </scm>
 
-    <developers>
-        <developer>
-            <id>fbelzunc</id>
-            <name>Felix Belzunce Arcos</name>
-        </developer>
-    </developers>
-
     <properties>
         <maven.compiler.source>1.6</maven.compiler.source>
         <maven.compiler.target>1.6</maven.compiler.target>
```

Comment:
Bumping the version to 2.x fails the build during the generation of the javadoc:

{code}
{code}